include/mata/nfa.hh

```diff
@@ -918,11 +968,11 @@ inline bool is_universal(
 /**
  * @brief Checks inclusion of languages of two NFAs: @p smaller and @p bigger (smaller <= bigger).
  *
- * @param smaller[in] First automaton to concatenate.
- * @param bigger[in] Second automaton to concatenate.
- * @param cex[out] Counterexample for the inclusion.
- * @param alphabet[in] Alphabet of both NFAs to compute with.
- * @param params[in] Optional parameters to control the equivalence check algorithm:
+ * @param[in] smaller First automaton to concatenate.
+ * @param[in] bigger Second automaton to concatenate.
+ * @param[out] cex Counterexample for the inclusion.
+ * @param[in] alphabet Alphabet of both NFAs to compute with.
```

Comment:
An argument to remove alphabet is that even in the naive algo, where alphabet is used, it is better to use only the alphabet of the symbols common for both automata than the entire alphabet (passed as the parameter).